Week7/REVIEW.md

```diff
@@ -1,9 +1,196 @@
 # REVIEW JavaScript week 7
 
+## Git Workflow
+
+To be provided
+
+## Map, filter, reduce 
+
+The array methods **map()**, **filter()** and **reduce()** are best understood by looking at how they could be implemented if we were to write them ourselves. In the next few sections we will present simplified versions of the native implementations. We have prefixed the method names with `my` to distinguish them from the built-in versions.
+
+### Array#map\*
+
+The **map** method returns a new array where each element of the subject array is transformed by a user-supplied transformation (= _mapping_) function. 
+
```

Comment:
maybe you can describe a bit more what the function below does, (in a simpler way) and with a real life example, something people can actually relate to